paneldb/server/templates/baitsets.html

```diff
@@ -19,9 +19,32 @@ <h4>Upload a new bait set</h4>
       <textarea class="form-control" id="exampleFormControlTextarea1" rows="2" ></textarea>
     </div>
 
+
+    <!-- Genomic build dropdown -->
+    <div class="dropdown">
+        <a class="btn btn-info dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+         Genomic Build
+        </a>
+
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+            <a class="dropdown-item" href="#">GRCh37</a>
+            <a class="dropdown-item" href="#">GRCh38</a>
+        </div>
+    </div>
+
+    <!-- testing
```

Comment:
Cars? :)

Reply:
Just tried to see if this dropdown worked